### PR TITLE
Update color token

### PR DESCRIPTION
**Pull request by jgagne**
## What this PR does

- Updates button hovered/focused state(s) color from gray (disabled) to a shade of blue 

### Visual Preview

- [Color tokens](https://toolness.github.io/accessible-color-matrix/?n=text&n=hovered%2Ffocused%20(current)&n=hovered%2Ffocused%20(proposed)&n=enabled&v=FFFFFF&v=444444&v=015894&v=0077C8) (comparison and color contrast)

**Before:**

gym-hover-focus-color-token-before

**After:**

gym-hover-focus-color-token-after

### CODE REVIEW

File: src/scss/mfe/learning/_buttons.scss [hunk 1]
```diff
@@ -37,8 +37,8 @@
   }
 
   &-outline-primary {
-    color: var(--primary);
-    border-color: (var(--primary));
+    color: var(--primary-500);
+    border-color: (var(--primary-500));
```
Comment (rediris) [inline src/scss/mfe/learning/_buttons.scss]: This rule looks a little wonky...

File: src/scss/mfe-discussions.scss [hunk 2]
```diff
@@ -447,7 +447,7 @@ header {
 
 .btn-icon.btn-icon-primary:hover {
   background-color: $light-300 !important;
-  color: var(--primary)
+  color: var(--primary-500)
```
Comment (rediris) [inline src/scss/mfe-discussions.scss]: needs closing semicolon

File: src/scss/mfe/_buttons.scss [hunk 3]
```diff
@@ -38,8 +38,8 @@
   }
 
   &-outline-primary {
-    color: var(--primary);
-    border-color: (var(--primary));
+    color: var(--primary-500);
+    border-color: (var(--primary-500));
```
Comment (rediris) [inline src/scss/mfe/_buttons.scss]: fix parentheses in line 42

File: src/scss/mfe/_forms.scss [hunk 4]
```diff
@@ -22,15 +22,15 @@ $background-color_4: #178253e6;
 $background-color_5: #1fad6f;
 $background-color_6: #c32d3ae6;
 $background-color_7: #d64d59;
-$background-color_8: var(--primary);
-$background-color_9: rgba(var(--primary), 0.8); // transparent-ish primary color?
+$background-color_8: var(--primary-500);
+$background-color_9: rgba(var(--primary-500), 0.8); // transparent-ish primary color?
```
Comment (rediris) [inline src/scss/mfe/_forms.scss]: I can't remember - does this work? We might be able to use `hsla` instead...

I was thinking maybe we can create a new transparent color map specifically for this, but maybe it's beyond the scope of this update.

File: src/scss/mfe/discussions/_default.scss [hunk 5]
```diff
@@ -556,9 +556,9 @@ a.muted-link:hover {
 }
 a.muted-link.inline-link,
 p > a.muted-link[href]:not(.btn) {
-  color: var(--primary);
+  color: var(--primary-500);
   text-decoration: underline;
-  text-decoration-color: var(--primary)4d;
+  text-decoration-color: var(--primary-500)4d;
```
Comment (rediris) [inline src/scss/mfe/discussions/_default.scss]: This one was "broken" before this change - this color needs to be another transparent color..., but I have no idea where it's used.

File: src/scss/mfe/discussions/_default.scss [hunk 6]
```diff
@@ -578,9 +578,9 @@ a.brand-link:hover {
 }
 a.brand-link.inline-link,
 p > a.brand-link[href]:not(.btn) {
-  color: var(--primary);
+  color: var(--primary-500);
   text-decoration: underline;
-  text-decoration-color: var(--primary)4d;
+  text-decoration-color: var(--primary-500)4d;
```
Comment (rediris) [inline src/scss/mfe/discussions/_default.scss]: Another broken var.

File: src/scss/mfe/discussions/_default.scss [hunk 7]
```diff
@@ -13423,7 +13423,7 @@ a.badge-primary:hover {
 }
 a.badge-primary.focus,
 a.badge-primary:focus {
-  box-shadow: 0 0 0 1px var(--primary)80;
+  box-shadow: 0 0 0 1px var(--primary-500)80;
```
Comment (rediris) [inline src/scss/mfe/discussions/_default.scss]: Another broken.

File: src/scss/mfe/discussions/_default.scss [hunk 8]
```diff
@@ -19978,7 +19978,7 @@ textarea.form-control {
 .custom-checkbox
   .custom-control-input:disabled:indeterminate
   ~ .custom-control-label:before {
-  background-color: var(--primary)80;
+  background-color: var(--primary-500)80;
```
Comment (rediris) [inline src/scss/mfe/discussions/_default.scss]: Another broken

File: src/scss/mfe/discussions/_default.scss [hunk 9]
```diff
@@ -19989,7 +19989,7 @@ textarea.form-control {
 .custom-radio
   .custom-control-input:disabled:checked
   ~ .custom-control-label:before {
-  background-color: var(--primary)80;
+  background-color: var(--primary-500)80;
```
Comment (rediris) [inline src/scss/mfe/discussions/_default.scss]: Another broken

File: src/scss/mfe/discussions/_default.scss [hunk 10]
```diff
@@ -20046,7 +20046,7 @@ textarea.form-control {
 .custom-switch
   .custom-control-input:disabled:checked
   ~ .custom-control-label:before {
-  background-color: var(--primary)80;
+  background-color: var(--primary-500)80;
```
Comment (rediris) [inline src/scss/mfe/discussions/_default.scss]: Another broken

File: src/scss/mfe/learning/_learning-original.scss [hunk 11]
```diff
@@ -553,9 +553,9 @@ a.muted-link:hover {
 }
 a.muted-link.inline-link,
 p > a.muted-link[href]:not(.btn) {
-  color: var(--primary);
+  color: var(--primary-500);
   text-decoration: underline;
-  text-decoration-color: rgba(var(--primary), 80);
+  text-decoration-color: rgba(var(--primary-500), 80);
```
Comment (rediris) [inline src/scss/mfe/learning/_learning-original.scss]: As above - I don't know if this works as expected...

File: src/scss/mfe/learning/_learning-original.scss [hunk 12]
```diff
@@ -5037,7 +5037,7 @@ a.badge-primary:hover {
 }
 a.badge-primary.focus,
 a.badge-primary:focus {
-  box-shadow: 0 0 0 1px var(--primary) 80;
+  box-shadow: 0 0 0 1px var(--primary-500) 80;
```
Comment (rediris) [inline src/scss/mfe/learning/_learning-original.scss]: broken rule

File: src/scss/mfe/learning/_learning-original.scss [hunk 13]
```diff
@@ -5065,7 +5065,7 @@ a.badge-brand:hover {
 }
 a.badge-brand.focus,
 a.badge-brand:focus {
-  box-shadow: 0 0 0 1px var(--primary) 80;
+  box-shadow: 0 0 0 1px var(--primary-500) 80;
```
Comment (rediris) [inline src/scss/mfe/learning/_learning-original.scss]: broken rule

File: src/scss/mfe/learning/_learning-original.scss [hunk 14]
```diff
@@ -7728,7 +7728,7 @@ textarea.form-control {
 .custom-checkbox
   .custom-control-input:disabled:indeterminate
   ~ .custom-control-label:before {
-  background-color: var(--primary) 80;
+  background-color: var(--primary-500) 80;
```
Comment (rediris) [inline src/scss/mfe/learning/_learning-original.scss]: broken rule

File: src/scss/mfe/learning/_learning-original.scss [hunk 15]
```diff
@@ -7739,7 +7739,7 @@ textarea.form-control {
 .custom-radio
   .custom-control-input:disabled:checked
   ~ .custom-control-label:before {
-  background-color: var(--primary) 80;
+  background-color: var(--primary-500) 80;
```
Comment (rediris) [inline src/scss/mfe/learning/_learning-original.scss]: Another broken

File: src/scss/mfe/learning/_learning-original.scss [hunk 16]
```diff
@@ -7783,7 +7783,7 @@ textarea.form-control {
 .custom-switch
   .custom-control-input:disabled:checked
   ~ .custom-control-label:before {
-  background-color: var(--primary) 80;
+  background-color: var(--primary-500) 80;
```
Comment (rediris) [inline src/scss/mfe/learning/_learning-original.scss]: Another broken

**Review by rediris — CHANGES_REQUESTED** [review]
This looks really good, and I like the color changes.

I've annotated some broken rules (broken before your update) that all correspond to places where openedx use 8 digit hex rules to represent transparency. I think the best solution is for us to create a new transparency color map and generate hsla values from it. Alternately, save it for later in another ticket, assuming we can figure out a way to best document the broken rules.

Let me know your thoughts.

**Comment by rediris** [comment]
> This looks really good, and I like the color changes.
> 
> I've annotated some broken rules (broken before your update) that all correspond to places where openedx use 8 digit hex rules to represent transparency. I think the best solution is for us to create a new transparency color map and generate hsla values from it. Alternately, save it for later in another ticket, assuming we can figure out a way to best document the broken rules.
> 
> Let me know your thoughts.

Oops - I didn't notice this was a WIP - sorry I jumped on this too soon.

**Comment by jgagne** [comment]
@rediris — Thanks for the pre-review. Keep you posted on updates.

**Review by rediris — APPROVED** [review]
Let's do it!

**Pull request merged**